cherry_picker/cherry_picker.py

```diff
@@ -843,23 +849,33 @@ def validate_sha(sha):
         )
 
 
-def version_from_branch(branch):
+def version_sort_key(config, branch):
```

Comment:
Can we add verb in here?
```suggestion
def compute_version_sort_key(config, branch):
```